presto-product-tests/src/main/resources/csv/data.csv

```diff
@@ -0,0 +1,3 @@
+header1,header2,header3
+value1,value2,value3
+footer1,footer2,footer3
```

Comment:
footer and header does not need to have same number of columns. It is a bit misleading because it looks like data.

```
#### HEADER #######
value1,value2,value3
#### FOOTER #######
```
?